opal/util/bipartite_graph.c

```diff
@@ -817,7 +817,9 @@ static int min_cost_flow_ssp(opal_bp_graph_t *gx, int **flow_out)
     return err;
 
 out_error:
-    free(*flow_out);
```

Comment:
flow_out cannot be NULL there is a check preventing that at the beginning of the function.

Reply:
Now that this PR removed `__opal_attribute_nonnull__` from the `opal_free()` wrapper, you're right.  Removed.